src/main/cli/settings.c

```diff
@@ -277,9 +277,8 @@ static const char * const lookupTableCameraControlMode[] = {
 
 static const char * const lookupTablePwmProtocol[] = {
     "OFF", "ONESHOT125", "ONESHOT42", "MULTISHOT", "BRUSHED",
```

Comment:
If looks like "OFF" has always been wrong and should be "PWM". Index 0 correlates to `PWM_TYPE_STANDARD` and it's represented as "PWM" in the Configurator.

Reply:
Yep, good point. The problem is that this will make the `diff` based backup / restore of the ~800 users who are using this protocol - I guess we'll just have to bite the bullet and add this to the release notes.